addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -1974,6 +1983,43 @@ QUnit.test('focusing a chat window of a chat should make new message separator d
     );
 });
 
+QUnit.test('chat window: click on correspondent name in chat header redirects to user form', async function (assert) {
+    assert.expect(4);
+
+    this.data['res.partner'].records.push({ id: 7, name: "Demo" });
+    this.data['mail.channel'].records.push({
+        channel_type: "chat",
+        is_minimized: true,
+        members: [this.data.currentPartnerId, 7],
+    });
+    const partnerFormDeferred = makeTestPromise();
+    const bus = new Bus();
+    bus.on('do-action', null, payload => {
+        assert.step('action_open_partner_form');
+        assert.deepEqual(payload.action, {
+            res_id: 7,
+            res_model: "res.partner",
+            type: "ir.actions.act_window",
+            views: [[false, "form"]]
+        }, "opened form view should be the form view displaying the clicked correspondent (partner 7)");
+        return partnerFormDeferred.resolve();
+    });
+    await this.start({ env: { bus } });
+    const chatWindowHeaderName = document.querySelector('.o_ChatWindowHeader_name');
+    assert.hasClass(
+        chatWindowHeaderName,
+        'o-clickable',
+        "should have o-clickable class when the thread has one correspondent"
```

Comment:
`o-clickable` class is a technical detail, which is just something convenient in DOM to mean this part is clickable.
Comment assertion should be meaningful enough just from a functional PoV. Example: _"name of thread in header part of chat window should be clickable when thread has a single correspondent"_.